bblfsh/pyuast.c

```diff
@@ -14,77 +14,231 @@ static const char *ReadStr(const void *data, const char *prop)
   return PyUnicode_AsUTF8(attribute);
 }
 
-static int ReadLen(const void *data, const char *prop)
-{
+static int ReadLen(const void *data, const char *prop) {
   PyObject *node = (PyObject *)data;
   PyObject *children_obj = PyObject_GetAttrString(node, prop);
-  if (!children_obj)
+  if (!children_obj) {
     return 0;
+  }
   return PySequence_Size(children_obj);
 }
 
-static const char *InternalType(const void *node)
-{
+static const char *InternalType(const void *node) {
   return ReadStr(node, "internal_type");
 }
 
-static const char *Token(const void *node)
-{
+static const char *Token(const void *node) {
   return ReadStr(node, "token");
 }
 
-static int ChildrenSize(const void *node)
-{
+static int ChildrenSize(const void *node) {
   return ReadLen(node, "children");
 }
 
-static void *ChildAt(const void *data, int index)
-{
+static void *ChildAt(const void *data, int index) {
```

Comment:
I know it was that way before, but some interface functions use `*node` as parameter and others use `*data`.

Reply:
Well, since I started with style fixes I should also fix that =)